wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Data.kt

```diff
@@ -6,6 +6,7 @@ import org.stellar.sdk.Memo
 import org.stellar.walletsdk.ApplicationConfiguration
 import org.stellar.walletsdk.util.Util.isHex
 
+@Suppress("ConstructorParameterNaming")
```

Comment:
I think you should just constructor parameters

Reply:
Can we use something like `@SerialName("min_amount") val minAmount` for all of them?

Reply:
Yes